sharc/topology/topology_input_map.py

```diff
@@ -10,18 +10,42 @@
 import matplotlib.pyplot as plt
 import matplotlib.patches as patches
 import pandas as pd
+from itertools import product
+from shapely.geometry import Point, Polygon
 
 from sharc.topology.topology import Topology
 from sharc.parameters.parameters_imt import ParametersImt
+from sharc.map.topography import Topography
 
 
 class TopologyInputMap(Topology):
     """
     Generates the coordinates of the BSs based on the base station physical cell data input file.
     """
-
-    def __init__(self, param: ParametersImt):
+    # TODO: topography type hint
```

Comment:
Isn't the type hint already set?

Reply:
Yes, it is. That `TODO` is unnecessary and was removed in commit 757d4098ee746558126086ba8b184bdcdf9e6b7a